scss/tech.scss

```diff
@@ -0,0 +1,34 @@
+
+.border{
+width: 200px;
+height: 300px;
+border: solid 1px;
+border-radius: 10px;
+border-color: #bed23c;
+}
+
+.tech{
+background-color: #bed23c;
```

Comment:
Check indentation, this should have two spaces at the left, the same with all elements contained